### ISSUE TITLE
Kerberos Constrained Delegation

### ISSUE THREAD

**Issue by RobinBeismann** [Badgerati/Pode#667]
### Describe the Feature
Pode should deliver the WinIdentity to make Kerberos Constrained Delegation possible as this allows to use the User Identity against a backend service like a SQL DB with Windows Integrated Authentication (WIA).

### Additional Context
Additionally the documentation should feature and example on how to impersonate.

I will implement that.

**Comment by RobinBeismann** [comment]
Hey @Badgerati,

I got the impersonation working using constrained Kerberos Delegation, quite easy actually.
However we'd need to give the WinIdentity back on the normal User Hashtable in the $WebEvent as we can't seem to be able to retrieve it another time.
I'll raise an issue and a pull request for it, I'm just wondering if I shall add a switch so we only pass the WinIdentity as Attribute back if it is specified or if we should push it at all times?

I personally don't think there's a huge security issue if we pass it back at every time as the Developer will have access to the Pode Source anyway.
Additionally the AD Admin needs to configure a Constrained Delegation per SPN for it to work.
I've restricted our IIS Instances to use Kerberos Authentication, I could imagine that it works without Delegation if NTLM is enabled.
